packages/utils/src/lib/reports/utils.ts

```diff
@@ -97,9 +97,21 @@ export function severityMarker(severity: 'info' | 'warning' | 'error'): string {
   return 'ℹ️';
 }
 
+const MIN_NON_ZERO_RESULT = 0.1;
+
+export function roundValue(value: number): number {
+  const roundedValue = Math.round(value * 10) / 10; // round with max 1 decimal
+  if (Math.abs(value) < 1 && value !== 0) {
+    return roundedValue === 0
+      ? MIN_NON_ZERO_RESULT * Math.sign(value)
+      : roundedValue;
+  }
```

Comment:
:sewing_needle: Nitpick - I think as single condition would be a bit easier to follow:

```suggestion
  if (Math.abs(value) < 1 && value !== 0 && roundedValue === 0) {
    return MIN_NON_ZERO_RESULT * Math.sign(value);
  }
```

In fact, I'm not sure that checking if it's in -1 to 1 range is necessary:

```suggestion
  if (roundedValue === 0 && value !== 0) {
    return MIN_NON_ZERO_RESULT * Math.sign(value);
  }
```

Reply:
You are absolutely right! That check was unnecessary